data/human/deep missions.txt

```diff
@@ -3365,19 +3361,20 @@ event "mani refit"
 
 
 mission "Mani Refit"
-	name "Mani Refit"
+	name "Mani Refitted"
 	to offer
 		has "event: mani refit"
-		has "Home for Skadenga 5: done"
 	description "Head back to <destination> and meet up with the Transport Union's Fleet."
 	source
-		government "Republic"
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not planet "Mani"
 	destination "Mani"
 	on offer
-		dialog
```

Comment:
I messed up and deleted my original message. 

If there's some rule against this, then I will, of course, submit to the rule. 
I don't think there's any issue though with there being a decline option. 
It's a message, 95% of what I do with E-mails is delete them.  
I would make the case that hitting that 'decline' button is tantamount to deleting it. 
Now, as to why you'd delete the mission at that point, well, it might be a bad idea or make little sense, but that's the player's prerogative. I'd argue that the vast majority of missions, and decline options in the game, doesn't make sense. Except in cases of illegal activities, or where you'd lock yourself out of a different story line, declining a mission only results in you losing access to content. 
I don't think fundamentally this situation is any different. I leave it to the player to make sensical, or non-sensical actions. I just give them, where I can, choice. 

With that in mind, I don't see there being any problem with them having the option to decline a message. But if there's a hard rule against it, I will accept it.

Reply:
Deleting an email is not the same thing as declining a job/mission, hence why the question is a matter of whether we want to allow the player to decline these missions.